website/content/en/releases/13.1R/relnotes.adoc

```diff
@@ -135,6 +148,9 @@ This section covers the boot loader, boot menu, and other boot-related changes.
 [[boot-loader]]
 === Boot Loader Changes
 
+UEFI boot is improved for amd64. The loader detects whether the loaded kernel can handle the in-place staging area (non-copying mode). The default is `copy_staging auto`. Auto-detection can be overridden, for example: with `copy_staging enable`, the loader will unconditionally copy the staging area to 2M, regardless of kernel capabilities. Also, the code to grow the staging area is more robust; for growth to occur, it's no longer necessary to hand-tune and recompile the loader. _(Sponsored by the FreeBSD Foundation)_
```

Comment:
@kostikbel this draft is paraphrased from the final paragraph of <https://www.freebsd.org/status/report-2021-07-2021-09/#_improved_amd64_uefi_boot>. Please, have I preserved the essence? 

In particular (original): 

> … detects if the loaded kernel can handle in-place staging area …

– becomes (here): 

> … detects whether the loaded kernel can handle _the_ in-place staging area …

– correct? 

----

To give us a sense of size and shape, for 13.0 the final was <https://www.freebsd.org/releases/13.0R/relnotes/#boot>. 

Thank you

Reply:
https://github.com/freebsd/freebsd-doc/commit/4f8240b0d997edc3ce0791b638f3b34081b38f89